quickstarts/post-configure-example/README.md

```diff
@@ -13,11 +13,11 @@ The steps described on this quickstart will work on the following container imag
 - Dashbuilder
 - RHPAM/DM Controller
 
-Note that, this procedure can be applied to any kid of post configure actions.
+Note that, this procedure can be applied to any kind of post configure actions.
```

Comment:
I just don't like the "that this" ting, and the comma is ungrammatical here, can we do just a "Note: This..."?

Reply:
make sense.